### PR TITLE
Refs #3809 - Remove various small cops

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

**Comment by ares** [comment]
LGTM

File: test/functional/api/v1/users_controller_test.rb [hunk 1]
```diff
@@ -137,22 +137,20 @@ def user_one_as_anonymous_viewer
...
-
-    as_user :one do
-      get :index, {}
-      assert_response :success
-    end
-
-    assert !User.find_by_login("admin").nil?
-  end
-=end
+#   test "should recreate the admin account" do
```
Comment (domcleal) [inline test/functional/api/v1/users_controller_test.rb]: Would you mind deleting this block?  I can categorically state that we do not any more :)

**Comment by dLobatog** [comment]
@domcleal done

**Comment by dLobatog** [comment]
Closed in favor of https://github.com/theforeman/foreman/pull/2174

**Pull request closed without merging**